### DISCUSSION TITLE
Присутствие Code review

### DISCUSSION THREAD

**Discussion by RV8V** [HowProgrammingWorks/LiveQA#60] [Q&A]
Если скинул ссылку на репозиторий проекта в разделе code review, студенту обязательно нужно присутствовать на звонке при процессе проверки или достаточно только ссылки?

**Comment by tshemsedinov** [comment]
Лучше присутствовать, но как получится

**Reply by RV8V** [reply]
Вы пропустили несколько проектов в самом начале для code review
